src/components/Scoremeter/index.tsx

```diff
@@ -19,18 +19,23 @@ const ScoreMeter = (props: ScoremeterProps) => {
         colorConfig = colorGuide[colorMode == 'light' ? 'lightComponents' : 'darkComponents']
             .scoremeter,
         scoreDesc,
-        showIndicators,
+        showIndicators = false,
         lowerLimit = 300,
         upperLimit = 900,
         showLegends = true,
     } = props;
 
-    const sanitizedReading = reading < 0 ? LOWER_THRESHOLD : reading;
     const sanitizedOldReading = oldReading
-        ? oldReading < 0
-            ? LOWER_THRESHOLD
+        ? oldReading < lowerLimit
```

Comment:
Please can you simplify the logic or wrap the truthy and falsy blocks in braces, to make it more readable. (I would advocate for simplification of logic) Thanks!

Something like: 

```js
const sanitizedOldReading = oldReading
        ? (oldReading < lowerLimit
            ? lowerLimit
            : (
                oldReading > upperLimit
                    ? upperLimit
                    : oldReading
            )
        )
        : LOWER_THRESHOLD;
```

Reply:
Eslint rules does not allow us to add braces for ternary expression. I had added braces but they got removed while committing the files